### DISCUSSION TITLE
Case insensitive terminal autocomplete

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#65] [Ideas]
So `.docker`, `Dustin` & `desktop.ini` would all be suggestions after typing `d` then tabbing

**Comment by hello-smile6** [comment]
Wait, `.docker` would be an option?

**Reply by DustinBrett** [reply]
Yes, as it is for cmd.exe, the `.` is ignored.

**Reply by hello-smile6** [reply]
Oh, bash is so much more logical.
